### ISSUE TITLE
5.57-90.0 -> MobiDB ERROR - One or more keys necessary for output format <0> are missing

### ISSUE THREAD

**Issue by jose-martinez-fignoni** [ebi-pf-team/interproscan#294]
Hi all! I am getting this ERROR with the latest version (5.57-90.0), but I am using the same `config.ini` file that was working for me in the previous version. Any clue where I should look? tks!

```
Command: python3 bin/mobidb/2.0/mobidb_lite.py /mnt/efs/genomes/results/SYM57343/WGS18118/ASM-txvdzN/annotation/ANN-79MFN4JTs/iprscan_822/temp/ip-172-60-37-232.ec2.internal_20221007_025458642_kvow//jobMobiDBLite/000000000001_000000000492.fasta

Error output from binary:

| ERROR    | One or more keys necessary for output format <0> are missing. Check your config.ini and make sure that all necessary keys are present. | MISSING KEY(S): ._disembl; ._espritz; ._iupred; ._globplot
```

The content from my `config.ini` file:
```[bin_directories]
# absolute path to the predictor folder
# ! DONT DELETE UNASSIGNED KEYS !
anchor=
disembl=
dynamine=
espritz=
fess=
globplot=
iupred=
jronn=
pfilt=
seg=
vsl2=

[thresholds]
mdb=0.625
anchor=0.5
dis465=0.5
disHL=0.086
dynamine_coil=0.5
espD=0.5072
espN=0.3089
espX=0.1434
iupl=0.5
iups=0.5
glo=0.0
jronn=0.5
pfilt=0.5
seg=0.5
vsl=0.5
fess_helix=0.5
fess_sheet=0.5
fess_coil=0.5

[outfmt_groups]
0=main
1=main
2=mobidb3
3=allid
```

**Comment by matthiasblum** [comment]
Hi @jose-martinez-fignoni,

The `config.ini` file included in InterProScan 5.57-90.0 should work and I would advise against using a config file from an older version (they should be identical anyway).

In your case, `[bin_directories]` is missing from the config file and that causes MobiDB-lite to fail finding the executables. Adding `[bin_directories]` at the top of the `config.ini` file should fix your issue.

**Comment by jose-martinez-fignoni** [comment]
Hi Matthias, sorry that was just my error when copying the contents of the file to this message, I do have the `[bin_directories]` line on top of it. Any other ideas about what might be happening here? Thank you!!
